docs/bidders/pubnative.md

```diff
@@ -0,0 +1,58 @@
+# Pubnative Bidder
+
+## Prerequisite
+Before adding PubNative as a new bidder, there are 3 prerequisites:
+- As a Publisher, you need to have Prebid Mobile SDK integrated.
+- You need a configured Prebid Server (either self-hosted or hosted by 3rd party).
+- You need to be integrated with Ad Server SDK (e.g. Mopub) or internal product which communicates with Prebid Mobile SDK.
+
+## Configuration 
+
+- app_auth_token is unique per publisher app. Please contact our account manager to obtain yours.
+- zone_id should be always set to 1 (unless special use case)
+- bidder should be always set to "pubnative"
```

Comment:
Do you think we could clarify that these values must be present in the `openrtb`'s json request `"ext"` field?  Maybe even explain a little further that the `zone_id` is an `int` and its value limits and the `app_auth_token` is a string. All of these are just ideas, of course.

Reply:
Update, PTAL 👍